Facial_Recognition_Models/Recognition Using LBP_SVM/Readme.md

```diff
@@ -0,0 +1,30 @@
+## Overview
+    Face detection and recognition using LBP features and Support Vector Machine.
+    This model first creates the rectangle and crop the facial area from image then extracts LBP features from image and pass it through SVM.
+
+## Dependencies
+    pip install numpy
+    pip install opencv-python
+    pip install sklearn
+    pip install skimage
+    
+    
+## Quick Start
+    1] git clone https://github.com/akshitagupta15june/Face-X.git
+    2] cd Recognition using LBP_SVM
```

Comment:
Same here, Change this accordingly to the current project Directory

Reply:
@SanjayMarreddi Updated readme file.